### DISCUSSION TITLE
Missing type in composite literal (typecheck)

### DISCUSSION THREAD

**Discussion by wesleymotaaraujo** [golangci/golangci-lint#3421] [Q&A]
Hey Guys, 

When I run the golangci-lint locally I don't get any errors, but on GitHub Actions, I'm getting this error message: 
```golang
Error: missing type in composite literal (typecheck)
```

![image](https://user-images.githubusercontent.com/98834916/207977484-aa62a04d-db24-4669-a8fe-de4e4a933e58.png)

Do you have any ideas on why this is happening?

**Comment by pascal-blokur** [comment]
Did you try running it with the docker image? I didn't get an error when running the binary, but an error did show up when I ran the docker image version.
`docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:v1.50.1 golangci-lint run -v`

**Comment by lightstar** [comment]
Had the same issue after upgrading. But later I noticed in documentation that "v3.0.0+ requires explicit setup-go installation step prior to using this action". So when I did just it (added setup-go action in the beginning of the workflow), the problem disappeared.

**Comment by nmccready** [comment]
Where do you disable `composite literal` in the config?
